modules/library/api/src/main/java/org/geotools/data/FeatureWriter.java

```diff
@@ -42,7 +42,7 @@
  * </li>
  * <li>
  * This API allows modification, and Filter based modification to be written.
- * Please see AbstractDataStore for examples of implementing common
+ * Please see ContentDataStore for examples of implementing common
  * opperations using this API.
```

Comment:
This interface is still used by ContentDataStore (for the low-level data access).